api/roles.go

```diff
@@ -31,9 +32,26 @@ func (c *Client) GetRoles(accessToken string, realmName string) ([]keycloak.Role
 	return resp, err
 }
 
+// GetRoles gets all roles for the realm or client with their attributes
+func (c *Client) GetRolesWithAttributes(accessToken string, realmName string) ([]keycloak.RoleRepresentation, error) {
```

Comment:
No sure of the name... briefRepresentation=false only adds Attributes or it adds other fields too?

Reply:
Yes, it only adds Attributes. But maybe we can change the name to GetFullRoles or GetFullRoleRepresentations ?

Reply:
As you want... if attributes are the only difference, it's ok